tampermonkey_scripts/README.md

```diff
@@ -9,3 +9,12 @@ the `Holdings` tab of the brokerage account that you want to copy from. When the
 the button will copy the holdings into JSON data to your clipboard which can then be pasted into
 the GoogleSheets or any other program that wants to use the JSON data.
 
+## etoro_holdings_getter.js
+
+This script will run on etoro.com/portfolio and place a grey `Copy Holdings` button next to navbar buttons. 
+When the button is clicked, the button will copy the holdings into JSON data to your clipboard which can then 
+be pasted into the GoogleSheets or any other program that wants to use the JSON data.
+
+###Known issues: 
+- currency won't calculate correct price
+- holdings from copies won't be included
```

Comment:
I'm not understanding this part, do you mean by copies like duplicates?

Reply:
Sorry, on etoro you can copy other people. Stocks copied won't show in the list but be hidden under the person you've copied.

Stock prices are not always in USD on etoro. Say f.ex. TEL.OL will be in NOK and BATS.L in GPX.

I will make an update to the text.